excel-importer/src/main/java/uk/gov/hmcts/ccd/definition/store/excel/validation/ChallengeQuestionValidator.java

```diff
@@ -0,0 +1,212 @@
+package uk.gov.hmcts.ccd.definition.store.excel.validation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.ccd.definition.store.domain.service.question.ChallengeQuestionTabService;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ChallengeQuestionDisplayContextParameterValidator;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationException;
+import uk.gov.hmcts.ccd.definition.store.domain.validation.ValidationResult;
+import uk.gov.hmcts.ccd.definition.store.excel.endpoint.exception.InvalidImportException;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.ParseContext;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.SpreadsheetParsingException;
+import uk.gov.hmcts.ccd.definition.store.excel.parser.model.DefinitionDataItem;
+import uk.gov.hmcts.ccd.definition.store.excel.util.mapper.ColumnName;
+import uk.gov.hmcts.ccd.definition.store.repository.entity.*;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.regex.Pattern;
+
+@Component
+public class ChallengeQuestionValidator {
+
+    private ParseContext parseContext;
+    private static final String ANSWER_MAIN_SEPARATOR = ",";
+    private static final String ANSWER_FIELD_SEPARATOR = "|";
+    private static final String ANSWER_FIELD_DOT_SEPARATOR = ".";
+    private static final String ANSWER_FIELD_ROLE_SEPARATOR = ":";
+    private static final String ANSWER_FIELD_MATCHER = "^\\$\\{\\S.{1,}.\\S.{1,}}$|^\\$\\{\\S.{1,}.\\S.{1,}}:\\[\\S{1,}\\]$";
+    private static final String ERROR_MESSAGE = "ChallengeQuestionTab Invalid";
+    private final ChallengeQuestionTabService challengeQuestionTabService;
+    private ChallengeQuestionDisplayContextParameterValidator challengeQuestionDisplayContextParameterValidator;
+
+    @Autowired
+    public ChallengeQuestionValidator(ChallengeQuestionTabService challengeQuestionTabService,
+                                      ChallengeQuestionDisplayContextParameterValidator challengeQuestionDisplayContextParameterValidator) {
+
+        this.challengeQuestionTabService = challengeQuestionTabService;
+        this.challengeQuestionDisplayContextParameterValidator = challengeQuestionDisplayContextParameterValidator;
+    }
+
+    public ChallengeQuestionTabEntity validate(ParseContext parseContext, DefinitionDataItem definitionDataItem) {
+
+        this.parseContext = parseContext;
+        validateQuestionId(definitionDataItem);
+        final String questionId = definitionDataItem.getString(ColumnName.CHALLENGE_QUESTION_QUESTION_ID);
+        Optional<ChallengeQuestionTabEntity> result = challengeQuestionTabService.findByQuestionId(questionId);
+        final ChallengeQuestionTabEntity challengeQuestionTabEntity = result.orElseGet(ChallengeQuestionTabEntity::new);
+        challengeQuestionTabEntity.setQuestionId(questionId);
```

Comment:
instead of creating the ChallengeQuestionTabEntity here, do this in parser.. validator should just validate the data from the sheet